src/components/image/image.test.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import {
+  testStyledSystemMargin,
+  testStyledSystemLayout,
+  testStyledSystemBackground,
+  testStyledSystemPadding,
+} from "../../__spec_helper__/test-utils";
+import Image from "./image.component";
+
+testStyledSystemMargin((props) => <Image {...props} />);
+testStyledSystemLayout((props) => <Image {...props} />);
+testStyledSystemBackground((props) => <Image {...props} />);
+testStyledSystemPadding((props) => <Image {...props} />);
+
+it("renders", () => {
```

Comment:
```suggestion
test("renders", () => {
```
**comment**: To stay consistent with other test files I would use `test`